aws/import_aws_security_group.go

```diff
@@ -9,9 +9,6 @@ import (
 	"github.com/terraform-providers/terraform-provider-aws/aws/internal/naming"
 )
 
-// Security group import fans out to multiple resources due to the
-// security group rules. Instead of creating one resource with nested
-// rules, we use the best practices approach of one resource per rule.
 func resourceAwsSecurityGroupImportState(
```

Comment:
Hi @spavuluri! In following with the convention we now more commonly use for single resource import, we can remove functionality described in this file and move to using the `schema.ImportStatePassthrough` reference in the `resource_aws_security_group.go` file's `Importer` field since only the GroupID is needed. At import time, the resource will still undergo the staterefresh via `SGStateRefreshFunc` in the `Read` function but the only additional step we need to add to the resource's `Read` function will be to set the `name_prefix` as done in this file via 
```
d.Set("name_prefix", aws.StringValue(naming.NamePrefixFromName(aws.StringValue(sg.GroupName)))

```